doc/src/benchmarking.todo

```diff
@@ -0,0 +1,82 @@
+
```

Comment:
This feature is unrefined and should remain undocumeted until I refactor it properly.